documentation/jetty-documentation/src/main/asciidoc/operations-guide/modules/module-eeN-webapp.adoc

```diff
@@ -0,0 +1,26 @@
+//
+// ========================================================================
+// Copyright (c) 1995-2022 Mort Bay Consulting Pty Ltd and others.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License v. 2.0 which is available at
+// https://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+// which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+// ========================================================================
+//
+
+[[og-module-eeN-webapp]]
+===== Module `{ee-all}-webapp`
+
+include::{JETTY_HOME}/modules/{ee-current}-webapp.mod[tags=description]
+
+If this module is not enabled, only Jetty-specific handlers may be deployed.
```

Comment:
@sbordet: I'm not entirely sure what "only Jetty-specific handlers" means in this line of the documentation. Is there another section of the docs where this use case is explained?

Reply:
No we need to write that section. For now remove this line, as it duplicates what's in the module description.